src/containers/Review/ReviewPageWrapper.tsx

```diff
@@ -53,21 +94,32 @@ const ReviewPageWrapper: React.FC = () => {
           subheader={strings.SUBTITLE_REVIEW}
         />
       </Container>
+
       <Form>
         {reviewSections.map((reviewSection) => {
-          const assignedToYou = reviewSection.assigned?.id === 6
+          const { id, firstName, lastName } = currentUser as User
+          const assignedToYou = reviewSection.assigned?.id === id
           return (
             <ReviewSection
               key={`Review_${reviewSection.section.code}`}
+              reviewer={`${firstName} ${lastName}`}
```

Comment:
Is this info used anywhere? What is the intention in formatting the reviewer name like this?

Reply:
This is displayed next to the Reviewer comment attached to the reviewResponse. Oh! I checked and the code seems to be gone too. I'll go after it now.

Reply:
Ok. It's back now: (Not looking pretty)
<img width="828" alt="Screen Shot 2021-01-14 at 2 12 38 PM" src="https://user-images.githubusercontent.com/16461988/104531326-cec1cd00-5672-11eb-859a-7d0606d8a7d1.png">